build.gradle

```diff
@@ -1,6 +1,7 @@
 // Dependencies
-def scalaMajorMinorVersion = "2.12"
-def scalaVersion = (String) "${scalaMajorMinorVersion}.9"
+def scalaVersionParts = System.getenv().getOrDefault('SCALA_VERSION', '2.12.9').tokenize(".")
```

Comment:
so sad ... but if that's the way go for it ...

Reply:
if you have any suggestions :disappointed: